app/src/main/java/com/github/pockethub/android/ui/PatchedScrollingViewBehavior.java

```diff
@@ -1,76 +0,0 @@
-/*
- * Copyright (c) 2015 PocketHub
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.github.pockethub.android.ui;
-
-import android.content.Context;
-import android.support.design.widget.AppBarLayout;
-import android.support.design.widget.CoordinatorLayout;
-import android.support.v4.view.ViewCompat;
-import android.util.AttributeSet;
-import android.view.View;
-
-import java.util.List;
-
-public class PatchedScrollingViewBehavior extends AppBarLayout.ScrollingViewBehavior {
```

Comment:
This is used, should not be removed. Used from `pager_with_tabs`, `pager_with_dots`, `welcome_activity`.